src/main/java/cc/polyfrost/oneconfig/internal/renderer/NanoVGHelperImpl.java

```diff
@@ -681,6 +681,10 @@ public void resetTransform(long vg) {
      */
     @Override
     public void setAlpha(long vg, float alpha) {
+        //FIXME: this would break a lot of stuff.
+        if (alpha > OneConfigGui.INSTANCE.transparencyFactor) {
+            alpha = OneConfigGui.INSTANCE.transparencyFactor;
+        }
```

Comment:
should we keep this as-is?

Reply:
make it do that if OneConfigGui is drawing, else set it to normal

Reply:
was thinking it would be a performance hit but probably not

Reply:
just check if the screen is OneConfigGui